### DISCUSSION TITLE
Auto register pages/routes

### DISCUSSION THREAD

**Discussion by cyberkid042** [quasarframework/quasar#11310] [Ideas / Proposals]
Hi all, I am new to quasar, and I must say thank you to the team for making a tool like this. 

I'd like to know it quasar has something similar to nuxt in terms of adding pages, and auto registering them.

Thanks.

**Comment by hawkeye64** [comment]
You could look into these:

https://github.com/ktsn/vue-router-layout
https://github.com/ktsn/vue-cli-plugin-auto-routing
https://github.com/ktsn/vue-route-generator
https://github.com/ktsn/vue-auto-routing
